tlslite/tlsconnection.py

```diff
@@ -4934,6 +4996,19 @@ def _calculate_master_secret(self, premaster_secret, cipher_suite,
                               output_length=48)
         return secret
 
+    def _log_session_keys(self, keys):
+        if isinstance(keys, tuple):
+            keys = [keys]
+
+        with open(self.sslkeylogfile, 'a') as ssl_key_log_file:
+            ssl_key_log_file.writelines(
+                "{0} {1} {2}\n".format(
+                    label,
+                    binascii.hexlify(client_random).decode().upper(),
```

Comment:
using `b2a_hex()` from `compat` would be better: https://github.com/tlsfuzzer/tlslite-ng/blob/c93df82603ab4a72c8d65645a3aeee73eeb24b3f/tlslite/utils/compat.py#L68